cloudsim-plus/src/main/java/org/cloudbus/cloudsim/power/models/PowerModelHostSimple.java

```diff
@@ -0,0 +1,57 @@
+package org.cloudbus.cloudsim.power.models;
+
+import org.cloudbus.cloudsim.power.PowerMeasurement;
+
+import static org.cloudbus.cloudsim.power.PowerMeasurement.validatePower;
+
+/**
+ * Simple power model for hosts with linear power profile.
+ * @since CloudSim Plus 6.0.0
+ */
+public class PowerModelHostSimple extends PowerModelHost {
+    private double maxPower;
+    private double staticPower;
+
+    /**
+     * Instantiates a {@link PowerModelHostSimple} by specifying its static and max power usage.
+     *
+     * @param maxPower power (in watts) the host consumes under full load.
+     * @param staticPower power (in watts) the host consumes when idle.
+     */
+    public PowerModelHostSimple(final double maxPower, final double staticPower) {
+        if (maxPower < staticPower) {
+            throw new IllegalArgumentException("maxPower has to be bigger than staticPower");
+        }
+
+        this.maxPower = validatePower(maxPower, "maxPower");
+        this.staticPower = validatePower(staticPower, "staticPower");
+    }
```

Comment:
About my concern of changing the meaning of staticPower from percentage to watts,
you said that since this is a new class, the researcher can take a look at the docs and realize the values
changed.
The issue is that he/she may miss this change and get incorrect simulations results.
My proposal was to use the Unit enum, but I think it makes things more difficult and it will be strange to apply the unit just for the staticPower parameter.

What I did instead was to thrown an exception if maxPower or staticPower is a positive number smaller than 1.
That may indicate the researcher was trying to provide a value in percentage, instead of watts.
There may be equipment that consumes a fraction of a watt, but maybe this won't be an issue in the short term.

What do you think?

Reply:
I 100% agree, I also thought about something like that. Hardware that consumes <1 Watt is unlikely and in the very rare case, the researcher can still implement their own version of `PowerModelHost` 👍